docs/content/en/docs/getting-started/production-environment/gs-mgmt-cluster.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Create production cluster"
+weight: 40
+---
+
+EKS Anywhere supports a vSphere provider for production grade EKS Anywhere deployments.
+EKS Anywhere allows you to provision and manage Amazon EKS on your own infrastructure.
+
+This document walks you through setting up EKS Anywhere in a way that:
+
+* Deploys a management cluster on your vSphere environment
+* Deploys one or more workload clusters from the management cluster
+* Keeps the management cluster in place so you can use it to later upgrade and delete workload clusters
+
+{{% alert title="Important" color="warning" %}}
+
+In the initial release, EKS Anywhere clusters functioned as both workload and management clusters.
+Separating management features into a separate, persistant management cluster
+provides a cleaner model for managing the lifecycle of workload clusters (to create, upgrade, and delete clusters), while workload clusters run user applications.
+This approach also reduces provider permissions for workload clusters.
+
+{{% /alert %}}
+
+## Prerequisite Checklist
+
+EKS Anywhere needs to be run on an administrative machine that has certain [machine
+requirements]({{< relref "../install" >}}).
+An EKS Anywhere deployment will also require the availability of certain
+[resources from your VMware vSphere deployment]({{< relref "/docs/reference/vsphere/vsphere-prereq/_index.md" >}}).
+
+## Steps
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate a management cluster config:
+   ```bash
+   CLUSTER_NAME=prod
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider vsphere > eksa-mgmt-cluster.yaml
+   ```
+
+1. Modify the management cluster config (`eksa-mgmt-cluster.yaml`):
```

Comment:
users can also generate new config using 

```
eksctl generate clusterconfig <cluster name> -p vsphere
```

Reply:
I think I have it now.